.github/workflows/test-python.yml

```diff
@@ -0,0 +1,80 @@
+name: Tests Python
+
+on:
+  push:
+    paths-ignore: ["docs/**"]
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+    paths-ignore: ["docs/**"]
+  workflow_dispatch:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        # TODO: Enable ubuntu-20.04
+        # runs-on: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
+        runs-on: [ubuntu-16.04, ubuntu-18.04]
+        python-version: ["3.7"]
+
+    runs-on: ${{ matrix.runs-on }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      # TODO: Extract apt dependencies in dependencies manager
+      #       and only install for analyzer
+      - name: Install dependencies
+        run: |
+          if [[ "${{ matrix.runs-on }}" ==  "ubuntu-18.04" ]]; then
+            sudo add-apt-repository -y ppa:libretime/libretime
```

Comment:
This is only needed for upgrading out-dated silan versions. So should also need to be run on `ubuntu-16.04`.